manifests/init.pp

```diff
@@ -75,6 +75,24 @@
 #
 # $candlepin_manage_db:: Boolean indicating whether a database should be installed, this includes db creation and user
 #
+# $enable_candlepin::   Whether to enable candlepin
+#
+# $enable_qpid::        Whether to enable qpid
+#
+# $enable_qpid_client:: Whether to enable qpid client
+#
+# $enable_pulp::        Whether to enable pulp
+#
+# $enable_application:: Whether to enable application (katello web ui)
```

Comment:
Not sure I like the `enable` vocabulary, these are not optional, just how you manage them is. So I'd lean towards more of the `manage_` paradigm like we use elsewhere.